CHANGELOG.md

```diff
@@ -54,6 +54,7 @@
 - Fix cleanup of removed triggers ([#1768](https://github.com/kedacore/keda/pull/1768))
 - Eventhub Scaler: Add trigger parameter `checkpointStrategy` to support more language-specific checkpoints ([#1621](https://github.com/kedacore/keda/pull/1621))
 - Fix Azure Blob scaler when using multiple triggers with the same `blobContainerName` and added a optional `metricName` field ([#1816](https://github.com/kedacore/keda/pull/1816))
+- Fix READY and ACTIVE fields of ScaledJob to show status when we run `kubectl get sj` ([#1855](https://github.com/kedacore/keda/pull/1855))
```

Comment:
This should be in the ## Unreleased section, KEDA 2.3.0 is already out :)

Reply:
oh, by mistake I wrote here, I am changing it now.
 sorry for inconvience.

Reply:
Addressed the comment. PTAL